File: app/src/main/java/ru/prsolution/winstrike/mvp/apimodels/PaymentResponse.java

package ru.prsolution.winstrike.mvp.apimodels;

import java.io.Serializable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PaymentResponse implements Serializable
{

    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("redirect_url")
    @Expose
    private String redirectUrl;
    private final static long serialVersionUID = 2905219164848985577L;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getRedirectUrl() {
        return redirectUrl;
    }

    public void setRedirectUrl(String redirectUrl) {
        this.redirectUrl = redirectUrl;
    }

}

File: app/src/main/kotlin/ru/prsolution/winstrike/datasource/model/PaymentResponse.kt
package ru.prsolution.winstrike.datasource.model

import java.io.Serializable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PaymentResponse : Serializable {

	@SerializedName("message")
	@Expose
	var message: String? = null
	@SerializedName("redirect_url")
	@Expose
	var redirectUrl: String? = null

	companion object {
		private const val serialVersionUID = 2905219164848985577L
	}

}
